plugins/coatils.py

```diff
@@ -110,7 +109,7 @@ def ls(self, msg, match):
         Example: `ls bears python python3`
         """
         langs = list(map(lambda x: x.lower(), match.group(1).split()))
-        all_langs = Coatils.all_langs()
+        Coatils.all_langs()
```

Comment:
the returned value wasnt being used, and invoking `Coatils.all_langs` doesnt have side-effects .
You can remove the line.

(whereas the `self.pop_message()` in `coatils_test.py` does have side effects and can not be removed.)